app/controllers/static_controller.rb

```diff
@@ -29,6 +29,14 @@ def about
   def jfdi_signup
     @full_form = !request.xhr?
     return redirect_to profile_path if current_user
+
+    # If the redirect_to_tt parameter exists, store the URL
+    # to redirect to after signup
+    # See https://github.com/plataformatec/devise/wiki/How-To:-Redirect-back-to-current-page-after-sign-in,-sign-out,-sign-up,-update
+    if params[:redirect_to_tt] && TeaTime.find(params[:redirect_to_tt])
+      store_location_for(:user, '/tea_times/' + params[:redirect_to_tt].to_i.to_s)
```

Comment:
the param should be coerced to a string automatically when you concatenate it to the result of `store_location_for` - does it not work without the `to_i.to_s` that way?

Reply:
Yeah, this was a bit of a weird design decision. I was thinking that it was possible that `params[:redirect_to_tt]` might not be a string containing an integer, so here I was ensuring that it was an integer, but if we do `TeaTime.find(params[:redirect_to_tt])` (which I added later) that already checks if it's an integer. I'll remove this.